activestorage/lib/active_storage/service/s3_service.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
-require "aws-sdk"
+begin
+  require "aws-sdk-s3"
+rescue LoadError
+  require "aws-sdk"
+end
```

Comment:
Given that Active Storage is still unreleased, I think it can just depend on v3 of the SDK. We can require the standalone S3 gem without a fallback.

Reply:
Agree, let's support only last version of aws-sdk-s3 👍

Reply:
:+1: I fixed.